donkeycarManagerFront/src/components/carView.vue

```diff
@@ -51,7 +51,7 @@
         <div v-if="job && job.screen_msg_display" class="end message">
           <format-message :message="job.screen_msg" />
         </div>
-        <div v-if="!race" class="no-race"> Veuillez avancer pour lancer la course </div>
+        <div v-if="!race && job && job.name != 'AI_ASSISTED'" class="no-race"> Veuillez avancer pour lancer la course </div>
```

Comment:
maybe add a icon for the `AI_ASSISTED` job like :
🧠 Voiture en conduite autonome or
🧠 Conduite autonome
🧠 Conduite autonome, plus besoin de pilote
or a joke like 
[tesla-meme](https://i.kinja-img.com/gawker-media/image/upload/c_fit,fl_progressive,q_80,w_636/orq2czwvtlyy1wwh4az4.gif)
🧠 La voiture prend le relais